<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>544</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainGen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_Editor">
       <property name="minimumSize">
        <size>
         <width>707</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="title">
        <string>Hauptprogramm</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="font">
           <font>
            <family>Lucida Console</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Rustplan">
       <attribute name="title">
        <string>Rüstplan</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="tableView_Rustplan">
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Magazin" name="tab_Magazin">
       <attribute name="title">
        <string>Magazin</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
      <widget class="C_Algin" name="tab_C_Algin">
       <attribute name="title">
        <string>C Ausrichten</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </widget>
      <widget class="QWidget" name="tab_Log">
       <attribute name="title">
        <string>Logfile</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit_Log">
          <property name="font">
           <font>
            <family>Lucida Console</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_Open_2"/>
    <addaction name="action_Export"/>
    <addaction name="action_Print"/>
    <addaction name="action_FinishFile"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menuDatenbank">
    <property name="title">
     <string>Datenbank</string>
    </property>
    <addaction name="action_RestoreDatabase"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuDatenbank"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="action_Print">
   <property name="text">
    <string>Drucken</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="action_FinishFile">
   <property name="text">
    <string>Finish File</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="action_Open_2">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="action_RestoreDatabase">
   <property name="text">
    <string>Wiederherstellen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C_Algin</class>
   <extends>QWidget</extends>
   <header>c_algin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Magazin</class>
   <extends>QWidget</extends>
   <header location="global">magazin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
